src/OverlayTrigger.js

```diff
@@ -212,18 +211,18 @@ class OverlayTrigger extends React.Component {
 
     const triggerProps = {};
 
-    let triggers = trigger == null ? [] : [].concat(trigger);
+    let triggers = trigger ? [].concat(trigger) : [];
 
-    if (triggers.indexOf('click') !== -1) {
+    if (triggers.includes('click')) {
```

Comment:
i'm not sure we can use`Array.prototype.includes` without modifying our transpilation settings; this transpiles as-is to

```js
    if (triggers.includes('click')) {
      triggerProps.onClick = this.handleClick;
    }
```

but `Array.prototype.includes` is not available on IE full stop https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

and i don't think we want to require our users to pull in polyfills

Reply:
While I would normally go for adjusting the transpilation, for this PR it totally makes sense to leave it as is then.